ADVANCED.md

```diff
@@ -258,6 +258,7 @@ It is updated indirectly by invoking an action which results in a new API reques
 | `autocompletedSuggestions`          | Object[String, Array[[Suggestion](./packages/react-search-ui/src/types/Suggestion.js)] | A keyed object of query suggestions. It's keyed by type since multiple types of query suggestions can be set here.                                                                                                                                                        |
 | `autocompletedSuggestionsRequestId` | String                                                                                 | A unique ID for the current autocompleted suggestion results.                                                                                                                                                                                                             |
 | `facets`                            | Object[[Facet](./packages/react-search-ui/src/types/Facet.js)]                         | Will be populated if `facets` configured in [Advanced Configuration](#advanced-configuration).                                                                                                                                                                            |
+| `rawResponse`         | Object[Response] | The response object received from the API |
```

Comment:
Could you please add `rawResponse` to `DEFAULT_STATE` in `SearchDriver`? https://github.com/elastic/search-ui/blob/master/packages/search-ui/src/SearchDriver.js#L29